test/suites/osconfig/clusterid.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation       Tests verifying MicroShift cluster ID functionality
+
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/oc.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${CLUSTERID_FILE}       /var/lib/microshift/cluster-id
+${CLUSTERID_NS}         kube-system
+
+
+*** Test Cases ***
+Compare Cluster ID From Namespace And File
```

Comment:
Do we need this test? The other one does that and more.
Regardless if this test runs as first or second, it duplicates checks from the other.